clmm/theory/ccl.py

```diff
@@ -50,101 +39,116 @@ class CCLCLMModeling(CLMModeling):
-        self.hdpm_dict = {'nfw': ccl.halos.HaloProfileNFW,
-                          'einasto': ccl.halos.HaloProfileEinasto,
-                          'hernquist': ccl.halos.HaloProfileHernquist}
+            "mean": "matter",
+            "critical": "critical",
+            "virial": "critical",
+        }
+        self.hdpm_dict = {
+            "nfw": ccl.halos.HaloProfileNFW,
+            "einasto": ccl.halos.HaloProfileEinasto,
+            "hernquist": ccl.halos.HaloProfileHernquist,
+        }
         self.cosmo_class = CCLCosmology
-        self.hdpm_opts = {'nfw': {'truncated': False,
-                                  'projected_analytic': True,
-                                  'cumul2d_analytic': True},
-                          'einasto': {'truncated': False},
-                          'hernquist': {'truncated': False,
-                                        'projected_analytic': True,
-                                        'cumul2d_analytic': True}}
+        self.hdpm_opts = {
+            "nfw": {
+                "truncated": False,
+                "projected_analytic": True,
+                "cumul2d_analytic": True,
+            },
+            "einasto": {"truncated": False},
+            "hernquist": {
+                "truncated": False,
+                "projected_analytic": True,
+                "cumul2d_analytic": True,
+            },
+        }
         self.cor_factor = _patch_rho_crit_to_cd2018(ccl.physical_constants.RHO_CRITICAL)
-        self.__mdelta_cor = 0.0 ## mass with corretion for input
-        #self.hdpm_opts['einasto'].update({'alpha': 0.25}) # same as NC default
+        self.__mdelta_cor = 0.0  ## mass with corretion for input
+        # self.hdpm_opts['einasto'].update({'alpha': 0.25}) # same as NC default
 
         # Set halo profile and cosmology
         self.mdef = None
         self.conc = None
         self.set_halo_density_profile(halo_profile_model, massdef, delta_mdef)
         self.set_cosmo(None)
 
-
     # Functions implemented by child class
 
-
     def _update_halo_density_profile(self):
-        """"updates halo density profile with set internal properties"""
+        """ "updates halo density profile with set internal properties"""
         # prepare mdef object
         self.mdef = ccl.halos.MassDef(self.delta_mdef, self.mdef_dict[self.massdef])
         # adjust it for ccl version > 2.6.1
-        if parse(ccl.__version__) >= parse('2.6.2dev7'):
+        if parse(ccl.__version__) >= parse("2.6.2dev7"):
             ccl.UnlockInstance.Funlock(type(self.mdef), "_concentration_init", True)
         # setting concentration (also updates hdpm)
-        self.cdelta = self.cdelta if self.hdpm else 4.0 # ccl always needs an input concentration
+        self.cdelta = self.cdelta if self.hdpm else 4.0  # ccl always needs an input concentration
 
     def _get_concentration(self):
-        """"get concentration"""
+        """ "get concentration"""
```

Comment:
Same to what Hsin mentioned above. Plus several more instances below.

Reply:
done